.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,114 @@
+name: Micro benchmarks
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled]
+    branches:
+      - main
+      
+  push:
+    branches:
+      - "*"
+jobs:
+ Check_Degdratation:
+    name: Criterion Benchmarks
+    if: contains(github.event.pull_request.labels.*.name, 'benchmark') || contains(github.event.head_commit.message, 'benchmark')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install Critcmp
+        run: cargo install critcmp 
+
+      - name: Benchmark changes
+        run: |
+          cargo bench --bench impl_path_string_for_evaluation_context -- --save-baseline new_branch
+          cargo bench --bench data_loader_bench -- --save-baseline new_branch
+          cargo bench --bench request_template_bench -- --save-baseline new_branch
+          cargo bench --bench json_like_bench -- --save-baseline new_branch
```

Comment:
do we need to run benchmarks here one-by-one? `cargo bench` run all the benchmarks by default

Reply:
sloved using  cargo bench --features criterion -- --save-baseline main_branch